NEWS.md

```diff
@@ -1,5 +1,39 @@
 # NEWS
 
+## 1.0.1 - 2025-02-21
+
+### Package distribution update
+
+In previous releases, Ubuntu packages were available only through the Groonga
+APT repository. Starting with this release, packages are also distributed via
+packages.groonga.org.
+
+While the groonga-nginx package itself remains unchanged, the underlying Groonga
+dependency is different. The packages from packages.groonga.org include Groonga
+built with Apache Arrow enabled. Which unlocks extra features such as static
+parallel index building.
+
+### Migration Notice for groonga-nginx
```

Comment:
Migration should only be required for Ubuntu users, so it is better to add that.

Reply:
fix: [dbc6986](https://github.com/groonga/groonga-nginx/pull/25/commits/dbc6986f5c3faade4d9bd7080d92e669fbeab948)